toponetx/transform/graph_to_cell_complex.py

```diff
@@ -0,0 +1,29 @@
+"""Methods to lift a graph to a cell complex."""
+
+__all__ = [
+    "homology_cycle_cell_complex",
+]
+
+
+import networkx as nx
+
+from toponetx import CellComplex
+
+
+def homology_cycle_cell_complex(G):
+    """Get the neighbor complex of a graph.
+
+    Parameters
+    ----------
+    G : networkx graph
+        Input graph.
+
+    Returns
+    -------
+    CellComplex
+        the cell complex obtained by adding the homology cycles of the graph
+    """
+    cycles = nx.cycle_basis(G)
+    cx = CellComplex(G)  # lift to graph
```

Comment:
Remove unnecessary comments: comments are hard to maintain because they are not unittested and they can become outdated. the code should be clean enough to be read without comments. if documentation is needed, then explanations should go in the docstrings.